### ISSUE TITLE
Mage Quarter Tower

### ISSUE THREAD

**Issue by Brecht98** [dalaranwow/dalaran-wow#5947]
Hi there, my username is Brecht and my game character’s name is Brechtt. I wanted to enter to tower in the Mage Quarter of Stormwind, however; every time if I try and enter my game crashes. Now I came to the point where if I want to play with my character I just can’t anymore because it’s stuck in that place if the crash/bug. I wanted to teleport but I was not even able to click on it or my game had already crashed. I tried a few times restarting the pc and game, but it keeps crashing as soon as I open up this character. Hope you’ll be able to help me with this issue/bug.

 
[//]: # (INFORMATION ABOUT TAGS)
[//]: # (The black and yellow tag is unique. There can only be one label assigned to them - one black and one yellow)
[//]: # (Priority - ... tags are assigned by team members)
[//]: # (Your issue will be reviewed by testers and the developers will fix it in the future.)
[//]: # (Only once the issue got the label [Fixed - On Live Server] you can find the fix on your Gameserver.)
[//]: # (You can increase the relevance of your issue by adding a thumb-up emote http://i.imgur.com/RFouVVi.png)

[//]: # (***************************************************************)
[//]: # (** !!!!!!!!!!!!!DON'T DELETE THIS TEMPLATE ELSE YOUR ISSUE WILL BE CLOSED!!!!!!!!!!!! **)
[//]: # (***************************************************************)

**Current behaviour**: Tell us what happens in detail (Where, When, What, Why, Who)

**Expected behaviour**: Tell us what should happen instead

**Steps to reproduce the problem**:

1. 
2. 
3. 

**Include proofs for this behaviour**

You can research this on several pages like http://wowhead.com or http://wowwiki.wikia.com or http://www.youtube.com/
Issues without proofs will be closed and tag On Hold will be added until the proof is provided.

**Include the ID for the game objects, npcs (creatures,pets,minions), spells, items, quests, instances, zones, achievements, skills**

You can research this on several pages like https://wowgaming.altervista.org/aowow/ or https://wowhead.com/

**Include Screenshots from the issue if necessary**

**Comment by Golganeth** [comment]
Sometimes addons or corrupt UI elements may cause issues like this one. Are you having the issue with just one character in that place or with all of them?

Disable all of your addons check if it works. If that was indeed the problem I'd suggest you to turn them on one by one, so you can easily detect which addon was giving you trouble.

If the issue persists:
- You can clear your Cache folder (this might fix small minor issues and it's pretty quick to do). Try also clearing your WTF folder, you will lose your settings but it can potentially fix the problem as a last option.
- At this point if nothing has worked, it's most likely that your client has corrupted files so you should re-install the game from scratch. You can get our game client [here](https://www.dalaran-wow.com/game-client/).

In the meantime, you can make a ticket with another character and we will move your character out of that position so you can keep playing. Keep in mind though that if you do nothing to fix the client-side problem, you might have the same problem if you walk there again.

Closing up.
